### PR TITLE
Add a test for gl-native issue #15139

### PR DESCRIPTION
**Pull request by tmpsantos**
Test reduction for https://github.com/mapbox/mapbox-gl-native/issues/15139

### CODE REVIEW

File: test/integration/render-tests/regressions/mapbox-gl-native#15139/style.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 550,
```
Comment (asheemmamoowala) [inline test/integration/render-tests/regressions/mapbox-gl-native#15139/style.json]: Does the test need to be this big?

Comment (tmpsantos) [reply test/integration/render-tests/regressions/mapbox-gl-native#15139/style.json]: I can make it smaller.

Comment (tmpsantos) [reply test/integration/render-tests/regressions/mapbox-gl-native#15139/style.json]: Done.

**Comment by tmpsantos** [comment]
Btw, this is what we get on GL Native with this test today.

expected

**Pull request merged**